packages/docs/pages/grid.tsx

```diff
@@ -18,142 +18,205 @@ const ExampleBox: React.FC = ({ children }) => (
+                          commodo anim commodo. Ullamco consectetur elit ullamco aliquip do id consectetur anim laborum.
+                        </ExampleBox>
+                      </GridItem>
+
+                      <GridItem>
+                        <ExampleBox>
+                          In sit dolore culpa reprehenderit tempor laborum. Sit anim voluptate pariatur irure amet
+                          mollit. Est mollit ad pariatur esse eu reprehenderit ut veniam. Aliquip anim mollit aliquip
+                          quis officia minim esse laboris proident fugiat cillum. Eiusmod culpa eu ea eiusmod nostrud.
+                        </ExampleBox>
+                      </GridItem>
+
+                      <GridItem>
+                        <ExampleBox>
+                          Ipsum adipisicing pariatur magna quis incididunt amet pariatur. Aliqua voluptate consequat ut
+                          elit ea dolore officia reprehenderit exercitation dolore eiusmod cupidatat quis. Ullamco ipsum
+                          do deserunt sunt ad deserunt ut. Elit non velit dolore ad est eu.
+                        </ExampleBox>
+                      </GridItem>
+                    </Grid>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </>
+              ),
+            },
+          ]}
+        />
+      </Panel>
+
+      <Panel header="Props" headerId="props">
+        <ContentRoutingTabs
+          id="props"
+          routes={[
+            {
+              id: 'grid',
+              title: 'Grid',
+              render: () => (
+                <GridPropTable
+                  renderPanel={false}
+                  inheritedProps={
+                    <>
+                      <BoxPropTable collapsible />
+                      <DisplayPropTable collapsible />
+                      <MarginPropTable collapsible />
+                      <PaddingPropTable collapsible />
+                    </>
+                  }
+                />
+              ),
+            },
+            {
+              id: 'grid-item',
+              title: 'GridItem',
+              render: () => (
+                <GridItemPropTable
+                  renderPanel={false}
+                  inheritedProps={
+                    <>
+                      <BoxPropTable collapsible />
+                      <DisplayPropTable collapsible />
+                      <MarginPropTable collapsible />
+                      <PaddingPropTable collapsible />
+                    </>
+                  }
+                />
+              ),
+            },
+          ]}
+        />
+      </Panel>
 
-      <PageNavigation items={items} />
+      <Panel header="Do's and Don'ts" headerId="guidelines">
+        <GuidelinesTable
+          recommended={[
+            <>
+              Use <Link href="https://css-tricks.com/snippets/css/complete-guide-grid/">CSS-Tricks complete guide</Link>{' '}
```

Comment:
We probably want this on the `Flex` PR too 🤦 
```suggestion
              Use <Link href="https://css-tricks.com/snippets/css/complete-guide-grid/" external target="_blank" rel="external nofollow noreferrer">CSS-Tricks complete guide</Link>{' '}
```